internal/pipe/scoop/scoop.go

```diff
@@ -101,13 +101,21 @@ func doRun(ctx *context.Context, client client.Client) error {
 	if ctx.Config.Release.Disable {
 		return pipe.Skip("release is disabled")
 	}
+
+	var commitFormat string
+	if ctx.Config.Scoop.CommitFormat == "" {
```

Comment:
this should be done in the `Default` method and should also use templates as @jthurman42 pointed out

Reply:
Moved to default and updated to template stings.